src/components/pagination-nav/pagination-nav.js

```diff
@@ -0,0 +1,200 @@
+      this.manage(on(this.selectorPageSelect, 'change', evt => this.handleSelectChange(evt)));
+    }
+  }
+
+  /**
+   * Get active page number
+   */
+  initActivePageNumber = () => {
+    let pageNum;
+    const activePageElement = this.element.querySelector(this.options.selectorPageActive);
+    if (activePageElement) {
+      pageNum = Number(activePageElement.dataset.page);
+    }
+    return pageNum;
+  };
+
+  /**
+   * Clear active page attributes
+   */
+  clearActivePage = evt => {
+    const selectorPageButtonArray = this.element.querySelectorAll(this.options.selectorPageButton);
+    selectorPageButtonArray.forEach(el => {
+      el.classList.remove(this.options.classActive, this.options.classDisabled);
+      el.disabled = false;
+      el.removeAttribute('data-active');
+      this.anchorAttributesHelper(el);
+    });
+    if (this.selectorPageSelect) {
+      const selectorPageSelectOptions = this.selectorPageSelect.options;
+      for (let i = 0; i < selectorPageSelectOptions.length; i++) {
+        selectorPageSelectOptions[i].removeAttribute('data-active');
+      }
+      if (!evt.target.matches(this.options.selectorPageSelect)) {
+        this.selectorPageSelect.classList.remove(this.options.classActive);
+        this.selectorPageSelect.value = '';
+      }
+    }
+  };
+
+  /**
+   * Add active state on click
+   */
+  handleClick = evt => {
+    this.clearActivePage(evt);
+    const selectorPageElementArray = this.element.querySelectorAll(this.options.selectorPageElement);
+    if (evt.target.matches(this.options.selectorPageButton)) {
+      this.activePageNumber = Number(evt.target.dataset.page);
+    }
+    if (evt.target.matches(this.options.selectorPageDirection)) {
+      if (evt.target.dataset.pageDirection === 'previous') {
+        this.activePageNumber -= 1;
+      }
+      if (evt.target.dataset.pageDirection === 'next') {
+        this.activePageNumber += 1;
+      }
+    }
+    const selectorPageTarget = selectorPageElementArray[this.activePageNumber - 1];
+    selectorPageTarget.dataset.active = true;
+    if (selectorPageTarget.tagName === 'OPTION') {
+      this.selectorPageSelect.value = this.activePageNumber;
+      this.selectorPageSelect.classList.add(this.options.classActive);
+    } else {
+      selectorPageTarget.classList.add(this.options.classActive, this.options.classDisabled);
+      selectorPageTarget.disabled = true;
+      if (selectorPageTarget.tagName === 'A') {
+        selectorPageTarget.setAttribute('tabIndex', -1);
+        selectorPageTarget.setAttribute('aria-disabled', true);
+      }
+    }
+    this.setPrevNextStates();
+  };
+
+  /**
+   * Handle select menu on change
+   */
+  handleSelectChange = evt => {
+    this.clearActivePage(evt);
+    this.activePageNumber = Number(evt.target.value);
+    const selectorPageSelectOptions = this.selectorPageSelect.options;
+    selectorPageSelectOptions[selectorPageSelectOptions.selectedIndex].dataset.active = true;
```

Comment:
Given we make `[data-active]` attribute configurable (with `[data-active="true"]` selector), let's define `attribActive` option.